<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T11:06:46.4282963+03:00" level="ERROR" thread="14" domain="/LM/W3SVC/8/ROOT-1-130690659899803555" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T11:53:19.0210234+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130690687824620763" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T12:36:47.7932367+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130690713921463417" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T12:41:52.3926588+03:00" level="ERROR" thread="13" domain="/LM/W3SVC/8/ROOT-1-130690716952956809" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T12:44:39.7112288+03:00" level="ERROR" thread="10" domain="/LM/W3SVC/8/ROOT-1-130690717833887195" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T12:49:40.0714085+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130690721656585841" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 46KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T12:50:35.5145796+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/8/ROOT-1-130690721656585841" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T12:53:58.8132077+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130690724203841536" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:03:44.4987069+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130690729976051687" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:05:15.2328966+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130690729976051687" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:06:34.1994133+03:00" level="ERROR" thread="16" domain="/LM/W3SVC/8/ROOT-1-130690729976051687" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\Repository\trunk\KPIproduction\KPIWeb\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:16:12.4143484+03:00" level="ERROR" thread="12" domain="/LM/W3SVC/8/ROOT-1-130690737543353143" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\22.13.13 13,13 - копия\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:48:57.2955112+03:00" level="ERROR" thread="6" domain="/LM/W3SVC/8/ROOT-1-130690757215082834" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\22.13.13 13,13 - копия\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:49:58.7752192+03:00" level="ERROR" thread="15" domain="/LM/W3SVC/8/ROOT-1-130690757215082834" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\22.13.13 13,13 - копия\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:53:48.662351+03:00" level="ERROR" thread="11" domain="/LM/W3SVC/8/ROOT-1-130690760090422288" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\22.13.13 13,13 - копия\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T13:54:55.0459319+03:00" level="ERROR" thread="7" domain="/LM/W3SVC/8/ROOT-1-130690760759982985" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\22.13.13 13,13 - копия\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T14:11:14.6591037+03:00" level="ERROR" thread="9" domain="/LM/W3SVC/8/ROOT-1-130690770588250759" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\22.13.13 13,13 - копия\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
<event logger="KPIWeb.LogWriter" timestamp="2015-02-22T14:11:55.2615869+03:00" level="ERROR" thread="8" domain="/LM/W3SVC/8/ROOT-1-130690771001139602" username="V-PC\V"><message>System.Threading.ThreadAbortException: Поток находился в процессе прерывания.    в System.Threading.Thread.AbortInternal()    в System.Threading.Thread.Abort(Object stateInfo)    в System.Web.HttpResponse.AbortCurrentThread()    в System.Web.HttpResponse.End()    в System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)    в System.Web.HttpResponse.Redirect(String url)    в KPIWeb.Account.Login.LogIn(Object sender, EventArgs e) в c:\Users\V\Desktop\22.13.13 13,13 - копия\KPIWeb\Account\Login.aspx.cs:строка 54KPIWeb.LogWriter  Void WriteError(System.Exception) Line Number: 0</message><properties><data name="log4net:HostName" value="V-PC" /></properties></event>
